packages/core/src/components/checkbox/test/indeterminate/checkbox.e2e.ts

```diff
@@ -0,0 +1,24 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/checkbox/test/indeterminate/index.html';
+
+test.describe('tds-checkbox', () => {
+  test('Checkbox indeterminate state', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    // Find the checkbox and label elements
+    const checkbox = page.locator('tds-checkbox');
+    const labelElement = page.locator('tds-checkbox', { hasText: 'Label' });
```

Comment:
A check for the contents of the label text is done here but not int the other tests. Do we need them there as well or is this one enough? 🙂

Reply:
Actually gonna change that, so it looks for the element correct in the shadow dom. Update coming.